adobe/aem-core-cif-components
# CIF-1081 - Client side product price loading does not respect multi store hierarchy configuration

## Description

* Added `StoreViewExporter` model to expose Magento store view in DOM.
* Updated `CommerceGraphqlAPI` library to pass the `Store` header accordingly.

## How Has This Been Tested?

* Updated unit tests.

## Verification
* Open a product or category page and verify that the client-side GraphQL request that retrieves pricing information contains a `Store: default` header.
* Configure the website to use a different storeview, e.g. `venia`. You can do that by setting the `cq:magentoStore` property on a page. Verify that the client-side pricing requests now send a `Store: venia` header.

## Types of changes




## Checklist:
